hasura/migrations/default/1675459146542_create_activities_table/up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "public"."activities" (
+    "id" bigserial NOT NULL,
+    "circle_id" bigint,
+    "organization_id" bigint NOT NULL,
+    "actor_profile_id" bigint,
+    "target_profile_id" bigint,
+    "contribution_id" bigint,
+    "epoch_id" bigint,
+    "user_id" bigint,
+
+    "action" varchar(100) NOT NULL,
+
+    "created_at" timestamptz NOT NULL DEFAULT now(),
+    "updated_at" timestamptz NOT NULL DEFAULT now(),
+
+  PRIMARY KEY ("id") ,
+  FOREIGN KEY ("circle_id") REFERENCES "public"."circles"("id") ON UPDATE CASCADE ON DELETE set null,
```

Comment:
If we are going to cascade (vs restrict) perhaps we should just delete this activity instead of orphan it? idk

Reply:
yeah, that sounds reasonable. I've updated this to cascade deletes.